### PR TITLE
Flair and Genji

### PR DESCRIPTION
**Pull request by Thunderclaww**

### CODE REVIEW

File: user_flair/hero_flair.scss [hunk 1]
```diff
@@ -626,4 +626,12 @@
 
 .flair-master-probius::before {
   background-position: 40px -2880px
+}
+
+.flair-genji::before {
+  background-position: 0 -2900px
+}
+
+.flair-master-genji::before {
```
Comment (yuna9) [inline user_flair/hero_flair.scss]: Since we don't have the master portrait flair yet, we should hold off on adding this rule.

**Comment by yuna9** [comment]
How does `reddit-logo-2` differ from `reddit-logo` and is it being used?

File: user_flair/hero_flair.scss (hunk 1 above)
Comment (Thunderclaww) [reply user_flair/hero_flair.scss]: I won't add it in the Edit Flair, so people can't select one yet anyway. I can remove the lines, but I'll probably add them back in in a few days anyway.

**Comment by yuna9** [comment]
Can you also edit your commit message to be more easily searchable later? I suggest:

```
Add Genji flair and banner

Also adds flair for Playing Ducks and changes the main logo.
```

You can use `git commit --amend` to edit the message, save, then force push: `git push -f`.

**Comment by Thunderclaww** [comment]
reddit-logo-2 is the 2.0 logo and actually what has been on the site for a while. Just hasn't been added to Github.

and sure, about the description.

File: user_flair/hero_flair.scss (hunk 1 above)
Comment (yuna9) [reply user_flair/hero_flair.scss]: I don't like adding rules if we won't use them, but I don't mind hugely. Leaving it in is fine by me.

**Comment by yuna9** [comment]
Aha okay, great about `reddit-logo-2`.

**Pull request closed without merging**